public/hyperswitch/index.html

```diff
@@ -1,108 +1,133 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <title>Loading...</title>
-
-    <script src="/defaultConfig.js"></script>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta charset="UTF-8" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@400;500;600;700&family=IBM+Plex+Sans:wght@300;400;500;600;700&display=swap"
-      rel="stylesheet"
-    />
-    <style>
-      html,
-      body {
-        font-family: "Inter", sans-serif;
-        font-weight: 400;
-        font-size: 16px;
-        -webkit-font-smoothing: antialiased;
-        padding: 0;
-        margin: 0;
-        height: 100%;
-        width: 100%;
-        display: flex;
-        flex-direction: column;
-      }
 
-      #app {
-        display: flex;
-        flex-direction: column;
-        flex: 1;
-      }
+<head>
```

Comment:
is it just formatting or something changed?

Reply:
formatting as well as some changes are done